front/Navbar.tsx

```diff
@@ -69,4 +69,4 @@ const Navbar = () => {
   )
 }
 
-export default Navbar
+export default Navbar
\ No newline at end of file
```

Comment:
Add a newline at the end of the file to conform to best practices and avoid potential issues with file concatenation and version control systems.